download/views.py

```
@@ -96,8 +117,20 @@ def app_stats(request, app_name):
+        release = app.release_set.all()
+        by_date_model = ReleaseDownloadsByDate
+    elif app.platform == 'service':
+        release = app.servicereleases_set.all()
+        by_date_model = ServiceReleaseDownloadsByDate
+    elif app.platform == 'web':
+        release = app.webbundlereleases_set.all()
+        by_date_model = WebBundleReleaseDownloadsByDate
+    else:
+        raise Http404
+
     for release in releases:
```

Ruff (0.16.2)

[error] Undefined name `releases`

(F821)